modules/installation-extend-edge-nodes-aws-local-zones.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/cluster-tasks.adoc
+
+ifeval::["{context}" == "installing-aws-localzone"]
+:localzone:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="installation-extend-edge-nodes-aws-local-zones_{context}"]
+= Creating user workloads in AWS Local Zones
+After you create an Amazon Web Service (AWS) Local Zone environment, and you deploy your cluster, you can use edge worker nodes to create a user workloads in Local Zone subnets. 
+
+After the `openshift-installer` creates the cluster, the installer automatically specifies a taint effect of `NoSchedule` to each edge worker node. This means that a scheduler does not add a new pod, or deployment, to a node if the pod does not match the specified tolerations for a taint. You can modify the taint for better control over how each node creates a workload in each Local Zone subnet.
+
+The `openshift-installer` creates the machine set manifest file and attaches `node-role.kubernetes.io/edge` and `node-role.kubernetes.io/worker` labels to each edge worker node that is located in a Local Zone subnet.
+
+.Prerequisites
+
+* You have an access to the OpenShift CLI (`oc`).
+* You deployed your cluster in a Virtual Private Cloud (VPC) with defined Local Zone subnets for extending regular worker nodes to edge worker nodes. 
+* You ensured that your machine set manifest file specifies `node-role.kubernetes.io/edge` for the `taint` parameter.
+
+.Procedure
+
+. Create a `deployment` resource YAML file for an edge worker node that operates in a Local Zone subnet. Ensure that you specify the correct tolerations for your edge worker node, so that the scheduler can schedule any pods with matching taints to the cluster.
++
+.Example of a configured `deployment` resource for an edge worker node that operates in a Local Zone subnet
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment <1>
+metadata:
+  name: <local_zone_name> <2>
+  namespace: <local_zone_namespace> <3>
```

Comment:
this could cause confusion, IMHO. according to the original example, it can be anything user-defined (free form). I would keep the "sample values" add update the legend that those values can be changed. Keeping "sample-..."  or something related (maybe "my-app...") could be more helpful (or maybe update from `<local_zone_name>` to `<user_workload_zoneName>`.
Reading that statement "<>" I felt like something mandatory related to Local zone deployment, and it's not. The intent is to provide an example to use. WDYT?

Reply:
Yeah, we try and use user-replaced values to avoid any scenarios where a user might assume values in an example is what they need to define in their config file. Users should be aware of their use in our docs as they are common ways to call out that the values are examples and need to be replaced. 

See https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values